server/app/api.py

```diff
@@ -50,20 +71,9 @@ def parse_json_field(field):
         return field
     return json.loads(field)
 
-def parse_json_list_field(field):
-    """
-    Parses field or returns appropriate boolean value.
 
-    :param field: (string)
-    :return: (string) parsed JSON
-    """
-    if not field[0] == '[':
-        if field == 'false':
-            return False
-        elif field == 'true':
-            return True
-        return field
-    return json.loads(field)
+parse_json_list_field = parse_json_field
```

Comment:
I'm guessing there was some reason for this separation - but merging it into one function seems ok.